packages/s/sqids.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "sqids",
+  "description": "Generate YouTube-like ids from numbers.",
+  "keywords": ["sqids", "hashids", "encode", "ids"],
+  "license": "MIT",
+  "homepage": "https://sqids.org/javascript",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/sqids/sqids-javascript.git"
+  },
+  "authors": [
+    {
+      "name": "sqids.org"
+    }
+  ],
+  "autoupdate": {
+    "source": "npm",
+    "target": "sqids",
+    "fileMap": [
+      {
+        "basePath": "cjs",
+        "files": ["*.js", "*.map", "*.ts"]
+      },
+      {
+        "basePath": "esm",
+        "files": ["*.js", "*.map", "*.ts", "package.json"]
+      },
+      {
+        "basePath": "dist",
+        "files": ["*.js", "*.map"]
+      }
```

Comment:
Files in these directories have the same names, so they're going to overwrite each other when pulled out of the directories. If you're wanting to include the cjs/esm versions alongside the dist version, I'd recommend leaving `dist` with a `basePath` so that we pull that up, but then use an empty `basePath` for the cjs/esm versions so they stay in their subdirectory.